### PR TITLE
Refactor: trending widget

### PR DESCRIPTION
**Pull request by josenriagu**
## Description

- connect latest profiles in trending widget
- refactor/update DS components

## Checklist

- [x] I have read the **README** document
- [x] I have read the **CONTRIBUTING** document
- [x] I have added tests to cover my changes
- [x] All new and existing tests passed
- [x] My commit message follows [Conventional Commit Guideline](https://conventionalcommits.org/)

### CODE REVIEW

File: ui/design-system-components/src/components/Entry/EntryBox/index.tsx [hunk 1]
```diff
@@ -212,10 +216,10 @@ const EntryBox: React.FC<IEntryBoxProps> = props => {
 
   return (
     <>
-      <div className={tw(`${error && 'bg-[#FFFDF1]'}`)} style={style}>
-        <div className={tw(`flex flex-row justify-between p-4 shrink-0`)}>
-          <a
-            className={tw(`flex min-w-0 no-underline`)}
+      <Box customStyle={`${error && 'bg-[#FFFDF1]'}`} style={style}>
```
Comment (SeverS) [inline ui/design-system-components/src/components/Entry/EntryBox/index.tsx]: don't we have this color defined?

Comment (josenriagu) [reply ui/design-system-components/src/components/Entry/EntryBox/index.tsx]: I'll take a look now and also fix other hard-coded colors in that file.

File: ui/apps/akasha/src/components/item-page/common/original-item.tsx [hunk 2]
```diff
@@ -153,9 +153,7 @@ export function OriginalItem({
         <EntryBox
           isRemoved={entryData?.isRemoved}
           entryData={entryData}
-          onClickAvatar={(ev: React.MouseEvent<HTMLDivElement>) =>
-            handleAvatarClick(ev, entryData?.author?.did?.id)
-          }
+          onClickAvatar={ev => handleAvatarClick(ev, entryData?.author?.did?.id)}
```
Comment (SeverS) [inline ui/apps/akasha/src/components/item-page/common/original-item.tsx]: can we do a function currying here?
```
const handleAvatarClick = (did: string) => (ev) => {ev.preventDefault() ... }
```
and then just use it like 
```
onClickAvatar = {handleAvatarClick(entryData?.author?.did.id)}
```

**Review by SeverS — CHANGES_REQUESTED** [review]
Can be merged after the fn currying change..

**Review by SeverS — DISMISSED** [review]

**Review by SeverS — APPROVED** [review]

**Pull request merged**